packages/observability/__tests__/server/providers/pino-provider.test.ts

```
@@ -0,0 +1,578 @@
+      (console as any).flush.mockImplementation((callback: any) => {
+        callback(new Error('Flush error'));
+      });
+
+      await provider.flush();
+
+      expect((console as any).flush).toHaveBeenCalled();
+    });
+
+    test('should handle logger without flush method', async () => {
+      // Remove flush method
+      delete (console as any).flush;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)